CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
+
```

Comment:
The lack of newlines was intentional because that's also how [keep a changelog](https://keepachangelog.com/en/1.0.0/) does it. Please remove them again.

Reply:
So sorry about that, I think my IDE auto formatted it

Reply:
Yeah, I thought as much ;)